### PR TITLE
Error handler api endpoint

### PR DESCRIPTION
**Pull request by johakoch**
Enabled `error_handler`s for an error type defined in both `endpoint` and `api` with expanding `*` wildcard error type.

---
<details>
    <summary>Reviewer checklist</summary>
    <ul>
        <li>Read PR description: a summary about the changes is required</li>
        <li>Changelog updated</li>
        <li>Documentation: docs/{Reference, Cli, ...}, Docker and cli help/usage</li>
        <li>Pulled branch, manually tested</li>
        <li>Verified requirements are met</li>
        <li>Reviewed the code</li>
        <li>Reviewed the related tests</li>
    </ul>
</details>

### CODE REVIEW

File: config/runtime/error_handler.go [hunk 1]
```diff
@@ -13,6 +12,11 @@ import (
 	"github.com/avenga/couper/handler"
 )
 
+var wildcardMap = map[string][]string{
```
Comment (malud) [inline config/runtime/error_handler.go:15]: Can we somehow create those mappings as follow-up generation task via `errors/generate` ?

**Comment by johakoch** [comment]
wait for https://github.com/avenga/couper/pull/464 (`error_handler` block)

File: config/runtime/error_handler.go (hunk 1 above)
Comment (johakoch) [reply config/runtime/error_handler.go:15]: Good idea. However, here we only need mappings for errors related to `api`/`endpoint.`

File: config/runtime/error_handler.go [hunk 2]
```diff
@@ -21,48 +25,65 @@ func newErrorHandler(ctx *hcl.EvalContext, opts *protectedOptions, log *logrus.E
...
-					Proxies:   h.Proxies,
-					ErrorFile: h.ErrorFile,
-					Requests:  h.Requests,
-					Response:  h.Response,
+		// this works if wildcard is the only "super kind"
+		if mappedKinds, mkExists := wildcardMap[ref]; mkExists {
+			// expand wildcard:
+			// * set wildcard error handler for mapped kinds, no error handler for this kind is already set
+			// * remove wildcard error handler for wildcard
+			if wcHandler, wcExists := handlersPerKind["*"]; wcExists {
```
Comment (alex-schneider) [inline config/runtime/error_handler.go:41]: I see here the handling of `error_handler "*" {...}`, but not for `error_handler {...}`. But it works. Why? :)

Comment (johakoch) [reply config/runtime/error_handler.go:41]: config/configload/error_handler.go:
```go
func newKindsFromLabels(block *hcl.Block) ([]string, error) {
...
	if len(allKinds) == 0 {
		allKinds = append(allKinds, errors.Wildcard)
	}
```

**Review by alex-schneider — APPROVED** [review]

**Review by alex-schneider — APPROVED** [review]

**Pull request merged**